### DISCUSSION TITLE
[治安局声明] 不要在绳网乱发奇怪的图片啊啊啊！

### DISCUSSION THREAD

**Discussion by ZhuYuan-148** [share121/inter-knot#714] [其他]
![Screenshot_20240810_151823](https://github.com/user-attachments/assets/4b75a36d-1742-48e1-b813-f7af47bf3b52)
本人刑侦特勤组朱鸢，今日调查绳网的时候无意看到某些用户上传的淫秽图片，被吓了一大跳所以下意识开枪打坏工作电脑……绳网并非法外之地，治安局会加紧审查，不会遗漏任何疑似违法的行为！
![Screenshot_20240810_151838](https://github.com/user-attachments/assets/9302cfdf-c60e-4e37-9c36-4d0ce9f37ec1)

**Comment by hzjwork** [comment]
支持

**Comment by cheeseRatRat** [comment]
支持

**Comment by niegong** [comment]
智齿

**Comment by bugmaibear** [comment]
朱鸢姐姐~莫非这个绳网，你就是站长？😚

**Comment by aiTunes-dev** [comment]
质吃

**Comment by GanlinSama** [comment]
治安局的人……也上绳网么？

**Comment by hashiqi21** [comment]
这个委托我...不是委托啊？我不接了

**Comment by SDCOM-0415** [comment]
绳网本来就是法外之地诶 (~~zzz世界观~~)

**Comment by Callkit-P0p** [comment]
智齿

**Comment by Jakobboy** [comment]
支持

**Comment by YUI-zzzz** [comment]
这个委托我接了！

**Comment by YiMie148** [comment]
治安管小姐我劝你不要多管闲事

**Comment by QingMingqwq** [comment]
等等，治安局的人也看绳网？治安官小姐你是不是不想干了？

**Comment by niegong** [comment]
支持✊✊✊

**Comment by ZhuYuan-148** [comment]
是调查！调查绳网！（不是自己想看…

**Reply by sishui-buyin** [reply]
真的假的？我读书少你别骗我

**Comment by aaa00073** [comment]
钛直尺了，那种东西可以去别的地方嘛，还是想保住社区环境滴

**Comment by Magic-gif** [comment]
支持整顿！不过朱鸢....很惊讶你会上绳网

**Comment by SDCOM-0415** [comment]
调查绳网？让我来调查调查你吧！

**Comment by Lanxey** [comment]
现在只能由管理员亲自删帖，只能靠大家自觉维护环境了

**Comment by ChevalierDeSangreal** [comment]
希望那些不法人士远离绳网，只靠管理员还是太勉强了。

**Reply by ZhuYuan-148** [reply]
是的是的

**Comment by ZhuYuan-148** [comment]
各位新艾利都市民，治安局鼓励整顿网络不良风气。绳网的秩序需要各位的协助，支持前来治安局举报身边的违法犯罪。

**Comment by ran-mitake-ag** [comment]
治安局的各位肯定很闲，都看上绳网了，v我500丁尼请我喝咖啡

**Reply by ZhuYuan-148** [reply]
好哦，来治安局我带你去喝丁曼特调

**Comment by Elysia-lover** [comment]
是可爱的女孩子，我们来交个朋友吧~

**Reply by ZhuYuan-148** [reply]
刑侦特勤组朱鸢，有什么能为您效劳的？

**Comment by keerdun** [comment]
朱鸢警官，能不能托你拍几张青衣前辈的照片，我会付报酬的，没别的意思，只是很敬仰罢了( ･ิω･ิ)

**Reply by ran-mitake-ag** [reply]
兄弟我也要

**Reply by ZhuYuan-148** [reply]
好哦，那来治安局给你们看？

**Reply by ran-mitake-ag** [reply]
窝似盗洞客，窝看不懂_aaa_

**Comment by Sushizhu** [comment]
永远支持新艾利都治安局

**Comment by HoshimiMiyabi120** [comment]
肯定，有时候我也会因为看到一些图片而感到困扰，不过我已经把克服这种困难制定成了新修行的一环，我希望你也可以尝试一下，就像偶尔可以舍弃蜜瓜的果肉，去尝试瓜皮拌出来的沙拉一样
